src/components/Level/staticLevels/index.js

```diff
@@ -0,0 +1,213 @@
+import { ReactComponent as OnePointRedDino } from '../../../icons/dinosaurs/red-one-point.svg';
+import { ReactComponent as TwoPointsRedDino } from '../../../icons/dinosaurs/red-two-points.svg';
+import { ReactComponent as ThreePointsRedDino } from '../../../icons/dinosaurs/red-three-points.svg';
+import { ReactComponent as FivePointsRedDino } from '../../../icons/dinosaurs/red-five-points.svg';
+import { ReactComponent as TenPointsRedDino } from '../../../icons/dinosaurs/red-ten-points.svg';
+
+import { ReactComponent as OnePointBlueDino } from '../../../icons/dinosaurs/blue-one-point.svg';
+import { ReactComponent as TwoPointsBlueDino } from '../../../icons/dinosaurs/blue-two-points.svg';
+import { ReactComponent as ThreePointsBlueDino } from '../../../icons/dinosaurs/blue-three-points.svg';
+import { ReactComponent as FivePointsBlueDino } from '../../../icons/dinosaurs/blue-five-points.svg';
+import { ReactComponent as TenPointsBlueDino } from '../../../icons/dinosaurs/blue-ten-points.svg';
+
+export const dinos = {
+    '1': { id: 1, Component: OnePointRedDino },
+    '2': { id: 2, Component: TwoPointsRedDino },
+    '3': { id: 3, Component: ThreePointsRedDino },
+    '4': { id: 4, Component: FivePointsRedDino },
+    '5': { id: 5, Component: TenPointsRedDino },
+    '6': { id: 6, Component: OnePointBlueDino },
+    '7': { id: 7, Component: TwoPointsBlueDino },
+    '8': { id: 8, Component: ThreePointsBlueDino },
+    '9': { id: 9, Component: FivePointsBlueDino },
+    '10': { id: 10, Component: TenPointsBlueDino },
+  }
+
+  export const legends = {
```

Comment:
The formatting is a bit off here (It's tabbed one over). Do you have prettier?

Reply:
'9': { id: 9, Component: FivePointsBlueDino },
'10': { id: 10, Component: TenPointsBlueDino },
      ^--- double digits threw it off.
      I do have prettier disabled at the moment. Wasn't sure if the team was using it and didn't want to have to deal with unintentional changes.